docs/examples/arrow_example.ipynb

```diff
@@ -105,88 +198,65 @@
-    "        pa.array([1, 1], type=pa.int8()),  # to_status\n",
-    "        pa.array([0.11, 0.15], type=pa.float64()),  # r1\n",
-    "        pa.array([0.12, 0.16], type=pa.float64()),  # x1\n",
-    "        pa.array([4.1e-05, 5.4e-05], type=pa.float64()),  # c1\n",
-    "        pa.array([0.1, 0.1], type=pa.float64()),  # tan1\n",
-    "        pa.array([0.01, 0.05], type=pa.float64()),  # r0\n",
-    "        pa.array([0.22, 0.06], type=pa.float64()),  # x0\n",
-    "        pa.array([4.1e-05, 5.4e-05], type=pa.float64()),  # c0\n",
-    "        pa.array([0.4, 0.1], type=pa.float64()),  # tan0\n",
-    "    ],\n",
-    "    names=(\n",
-    "        \"id\",\n",
-    "        \"from_node\",\n",
-    "        \"to_node\",\n",
-    "        \"from_status\",\n",
-    "        \"to_status\",\n",
-    "        \"r1\",\n",
-    "        \"x1\",\n",
-    "        \"c1\",\n",
-    "        \"tan1\",\n",
-    "        \"r0\",\n",
-    "        \"x0\",\n",
-    "        \"c0\",\n",
-    "        \"tan0\",\n",
-    "    ),\n",
-    ")\n",
-    "sources = pa.table(\n",
-    "    [\n",
-    "        pa.array([6], type=pa.int32()),  # id\n",
-    "        pa.array([1], type=pa.int32()),  # node\n",
-    "        pa.array([1], type=pa.int8()),  # status\n",
-    "        pa.array([1.0], type=pa.float64()),  # u_ref\n",
-    "    ],\n",
-    "    names=(\"id\", \"node\", \"status\", \"u_ref\"),\n",
-    ")\n",
-    "sym_loads = pa.table(\n",
-    "    [\n",
-    "        pa.array([7, 8], type=pa.int32()),  # id\n",
-    "        pa.array([2, 3], type=pa.int32()),  # node\n",
-    "        pa.array([1, 1], type=pa.int8()),  # status\n",
-    "        pa.array([0, 0], type=pa.int8()),  # type\n",
-    "        pa.array([1.0, 2.0], type=pa.float64()),  # p_specified\n",
-    "        pa.array([0.5, 1.5], type=pa.float64()),  # q_specified\n",
-    "    ],\n",
-    "    names=(\"id\", \"node\", \"status\", \"type\", \"p_specified\", \"q_specified\"),\n",
+    "nodes_dict = {\"id\": [1, 2, 3], \"u_rated\": [10500.0, 10500.0, 10500.0]}\n",
+    "\n",
+    "\n",
+    "lines_dict = {\n",
+    "    \"id\": [4, 5],\n",
+    "    \"from_node\": [1, 2],\n",
+    "    \"to_node\": [2, 3],\n",
+    "    \"from_status\": [1, 1],\n",
+    "    \"to_status\": [1, 1],\n",
+    "    \"r1\": [0.11, 0.15],\n",
+    "    \"x1\": [0.12, 0.16],\n",
+    "    \"c1\": [4.1e-05, 5.4e-05],\n",
+    "    \"tan1\": [0.1, 0.1],\n",
+    "    \"r0\": [0.01, 0.05],\n",
+    "    \"x0\": [0.22, 0.06],\n",
+    "    \"c0\": [4.1e-05, 5.4e-05],\n",
+    "    \"tan0\": [0.4, 0.1],\n",
+    "}\n",
+    "\n",
+    "sources_dict = {\"id\": [6], \"node\": [1], \"status\": [1], \"u_ref\": [1.0]}\n",
+    "\n",
+    "sym_loads_dict = {\n",
+    "    \"id\": [7, 8],\n",
+    "    \"node\": [2, 3],\n",
+    "    \"status\": [1, 1],\n",
+    "    \"type\": [0, 0],\n",
+    "    \"p_specified\": [1.0, 2.0],\n",
+    "    \"q_specified\": [0.5, 1.5],\n",
+    "}\n",
```

Comment:
since most of the people diving into this example notebook are already using pyarrow (or get it directly from the database), they most likely have `pa.table` or at least pyarrow-native data types. i would prefer if you can create a full table/record batch directly in pyarrow (like it was before). For the array, you may be able to use `from_arrays` (https://arrow.apache.org/docs/python/generated/pyarrow.FixedSizeListArray.html#pyarrow.FixedSizeListArray.from_arrays ) or even create a separate table just for the sake the asymmetric value.

alternatively, we can just say that "there's currently no simple zero-copy solution for asymmetric data but here's a solution with copy"

Reply:
I made this change to make it cleaner. Mentioning individual types is a bit messy.
It is upto the user how they wish to create the array. In the end, it should match the schema created in the helper function which is created from power_grid_meta_data.

Its similar reasoning as to how we have power flow example as
```
lines = initialize_array()
lines["id"] = [1,2,3]
```
instead of 
```
lines = initialize_array()
lines["id"] = np.array([1,2,3], dtype=np.int32())
```

Reply:
the danger with record batch is that it's a view of the data, not actual pyarrow data. it can be anything: contiguous, strided, fortran order, ... The whole point of this notebook is to show actual arrow data being loaded into the PGM.